src/test/java/seedu/address/model/timetable/util/TypicalLesson.java

```diff
@@ -0,0 +1,171 @@
+package seedu.address.model.timetable.util;
+
+import org.joda.time.LocalTime;
+
+import seedu.address.model.location.Location;
+import seedu.address.model.timetable.Lesson;
+import seedu.address.model.timetable.Module;
+import seedu.address.model.timetable.time.SchoolDay;
+
+
+/**
+ * Represents some of Typical Lessons.
+ */
+public class TypicalLesson {
+
+    public static final Lesson MONDAY_8AM_2HR_LESSON = new Lesson(new Module("CS2103T"),
+            new LocalTime(8, 0),
+            new LocalTime(10, 0),
+            SchoolDay.MONDAY,
+            new Location("NUS", 1.34, 103.7));
```

Comment:
Wait hold on, there is already a venues file.

Reply:
Just hardcoding for convenience at the moment. Testing `Lesson`, `Timetable` and `Scheduler` was more of a very high priority. But I understand your concern. I will most definitely rectify it in future PRs.

Reply:
Yeah at the very least, it should be a `private static final` or something along those lines.